index.html

```diff
@@ -0,0 +1,138 @@
+    </div>
+  </section>
+
+  <section class="second-sec">
+    <p class="program">Main Program</p>
+    <hr class="pro-hr">
+    <div class="program-container">
+      <div class="pro-item">
+        <img src="./Images/class.png" class="pro-icon" alt="Presentations">
+        <h3 class="pro-header">Presentations</h3>
+        <div class="pro-div">
+          <p class="pro-para">Listen to mental health advodates present their views on mental health according to specialty.</p>
+        </div>
+      
+      </div>
+        <div class="pro-item">
+          <img src="./Images/group(2).png" class="pro-icon" alt="Groups">
+          <h3 class="pro-header">Groups</h3>
+          <div class="pro-div">
+            <p class="pro-para ppone">Get into groups to have more intimate discussions and share ideas.</p>
+          </div>
+          
+        </div>
+          <div class="pro-item">
+            <img src="./Images/conversation.png" class="pro-icon" alt="Counselling">
+            <h3 class="pro-header">Counselling</h3>
+            <div class="pro-div">
+              <p class="pro-para pptwo">Get a chance to talk to a counsellor in the event that you feel the need to throughout the Summit</p>
+            </div>
+            
+          </div>
+            <div class="pro-item">
+              <img src="./Images/comment.png" class="pro-icon" alt="Feedback">
+              <h3 class="pro-header">Feedback</h3>
+              <div class="pro-div">
+                <p class="pro-para ppthree">Give feedback on how the session went and where you may need help adjusting.</p>
+              </div>
+            </div>
+              <div class="pro-item">
+                <img src="./Images/management.png" class="pro-icon" alt="Social">
+                <h3 class="pro-header">social</h3>
+                <div class="pro-div">
+                  <p class="pro-para ppfour">Network with other mental health professionals and widen the scope of your work.</p>
+                </div>
+
+              </div>
+    </div>
+    <div class="btn-div">
+      <button class="pro-btn">Join the MHA Summit</button>
+    </div>
+    <div class="btn-div">
+      <a href="#" class="pro-link">See the whole program</a>
+    </div>
+
+  </section>
+
+  <section class="third-sec">
+    <p class="sp-title">Featured Speakers</p>
+    <hr class="pro-hr">
+
+    <div class="speakers-container"></div>
+    
+  </section>
+  <section class="fourth-sec">
+    <div class="partner-div">
+      <h3 class="partner-h3">Partner</h3>
+      <hr class="partner-hr">
+    </div>
+    <div class="partners">
+      <h3>Revive</h3>
+      <h3>HealthPlus</h3>
+      <h3>WellBody</h3>
+      <h3>Soulful</h3>
+    </div>
+  </section>
+  <footer>
+    <div class="footer-div">
+      <img src="./Images/Logo3.png" class="tafara-img" alt="logo">
+      <p class="rights-p">2022 Tafara Designs. Some Rights Reserved.</p>
+    </div>
+  </footer>
```

Comment:
- [ ] Kindly remove the gap between the partner section and the footer so your design matches the original design:
<img width="947" alt="image" src="https://user-images.githubusercontent.com/77190203/157641842-e1a94906-cb01-4c58-a083-6616901435b8.png">